.azure-pipelines/templates/azdev_setup.yml

```diff
@@ -14,7 +14,7 @@ steps:
       chmod +x env/bin/activate
       . env/bin/activate
 
-      pip install -q azdev==0.1.28
+      pip install azdev==0.1.29
```

Comment:
I'd suggest we unpin `azdev` so that the latest version is always used.

Reply:
> I'd suggest we unpin `azdev` so that the latest version is always used.

I prefer to pin the version.